### PR TITLE
added button to copy dmg logs to clipboard

### PR DESCRIPTION
**Pull request by shnok**

### CODE REVIEW

File: LostArkLogger/Utilities/Sniffer.cs [hunk 1]
```diff
@@ -91,7 +91,7 @@ void ProcessPacket(List<Byte> data)
                     var pc = new PKTNewPC(payload);
                     var pcClass = Npc.GetPcClass(pc.ClassId);
                     if (!NameToClass.ContainsKey(pc.Name)) NameToClass[pc.Name] = pcClass + (NameToClass.ContainsValue(pcClass) ? (" - " + Guid.NewGuid().ToString().Substring(0, 4)) : "");
-                    IdToName[pc.PlayerId] = pc.Name + " (" + pcClass + ")";
+                    IdToName[pc.PlayerId] = pc.Name; // + " (" + pcClass + ")";
```
Comment (shalzuth) [inline LostArkLogger/Utilities/Sniffer.cs:94]: Is this removing the class name?

Comment (shnok) [reply LostArkLogger/Utilities/Sniffer.cs:94]: yes, i can revert it and add a regex in the generated string if that's a problem

Comment (shalzuth) [reply LostArkLogger/Utilities/Sniffer.cs:94]: yes, want to preserve class name for the logs

**Comment by shalzuth** [comment]
I'm also worried this might be abused/etc. - since the intent is to share it in game. We don't necessarily want to communicate the we're using dps meters in game, much like FFXIV

**Comment by shnok** [comment]
> I'm also worried this might be abused/etc. - since the intent is to share it in game. We don't necessarily want to communicate the we're using dps meters in game, much like FFXIV

I know what you mean. Being able to share logs with friends/guild is handy. In the past in other games, my guild always asked the dps logs since they didnt have the skills or were too lazy to install a dps meter. People should know the risks of sharing logs with strangers.

**Comment by shalzuth** [comment]
> 

Can we take out the regex that allows copying to the game? So it can still be shared via discord/etc.

**Comment by shnok** [comment]
> > 
> 
> Can we take out the regex that allows copying to the game? So it can still be shared via discord/etc.

Well it's not specifically made to be copied in the game, the regex just removes the class name from the username for shorter logs/better readability

**Pull request closed without merging**